src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/DataFixtures/ORM/WebTestFixtures.php

```diff
@@ -27,8 +27,8 @@
 
 class WebTestFixtures extends Fixture
 {
-    public const TEAMNAME_SURF = 'urn:collab:group:vm.openconext.org:demo:openconext:org:surf.nl';
-    public const TEAMNAME_IBUILDINGS = 'urn:collab:group:vm.openconext.org:demo:openconext:org:ibuildings.nl';
+    public const TEAMNAME_SURF = 'urn:collab:group:dev.openconext.local:demo:openconext:org:surf.nl';
+    public const TEAMNAME_IBUILDINGS = 'urn:collab:group:dev.openconext.local:demo:openconext:org:ibuildings.nl';
     public const TEAMNAME_ACME = 'demo:openconext:org:acme.com';
```

Comment:
How come sometimes the stem is sometimes added, and other times not?
Even if it's the default stem. If the default changes, it feels like things could break?

Should they not be saved consistently with or without the stem?

Reply:
The stem can change between environments. Thats why we configure them. For test, they should match the one set in the `.env.test` IIRC.

I think we now save the teamname without the stem right?

Reply:
The stem is also configured to later recognize the team/invide role name (urn). It should start with the stem. And then have the teamname proposed by us (based on the service name)